ee_plugin/processing/export_geotiff.py

```diff
@@ -22,11 +22,17 @@ class ExportGeoTIFFAlgorithm(QgsProcessingAlgorithm):
     OUTPUT = "OUTPUT"
 
     def initAlgorithm(self, config: dict) -> None:
-        raster_layers = [
-            layer.name()
-            for layer in Map.get_iface().mapCanvas().layers()
-            if layer.providerType() == "EE"
-        ]
+        try:
+            raster_layers = [
+                layer.name()
+                for layer in Map.get_iface().mapCanvas().layers()
+                if layer.providerType() == "EE"
+            ]
+        except Exception as e:
+            raster_layers = []
+            local_context.log(f"Failed to retrieve raster layers: {e}")
```

Comment:
Hmm... is this right?  I don't think we have a `log` method on `local_context`:

https://github.com/gee-community/qgis-earthengine-plugin/blob/765f4375e7b85de096cf85e5d54b86f26b34ab40/ee_plugin/logging.py#L14-L25

I think we want `logger = logging.getLogger(__name__); logger.error(...)`